rdflib/namespace.py

```diff
@@ -141,7 +141,7 @@ def __getattr__(self, name):
         return self.term(name)
 
     def __repr__(self):
-        return f"Namespace({self!r})"
+        return f"Namespace({self})"
```

Comment:
Can we make this:
```suggestion
        return f"Namespace({super().__repr__()})"
```
Or some other way to ensure the output URI is _quoted_ as a [Python expression](https://docs.python.org/3/reference/datamodel.html#object.__repr__).

Reply:
fixed and added tests for it.